direct/common/subsample_config.py

```diff
@@ -1,20 +1,23 @@
-# coding=utf-8
 # Copyright (c) DIRECT Contributors
+
+from __future__ import annotations
+
```

Comment:
Can you check until which version this is needed? It might be we have deprecated this.

Reply:
So as I understand its not needed for python > 3.8 (for typing) but needed for other stuff like future referencing. If you think I shouldn't keep it I will test if it works without it.

Reply:
I read a few things and as I understand starting with Python 3.10, type annotations are evaluated in a postponed manner by default, so this import is no longer necessary. So until we migrate to > 3.10 we keep it I think.